requirements.txt

```diff
@@ -8,3 +8,4 @@ dataclasses
 rich
 scikit-learn
 wandb
+submitit
```

Comment:
submitit is already here: https://github.com/facebookresearch/EGG/blob/master/nest.txt

We should just add to the readme that in order to use nest one should pip install nest.txt as well